### DISCUSSION TITLE
Set control panel "site--[SITE-HANDLE]" body class to mimic "site" querystring param for consistency

### DISCUSSION THREAD

**Discussion by richhayler** [craftcms/cms#11303] [Ideas] [answered]
For multi-site setups, a body class of `site--SITE-HANDLE` (based on the selected site) is applied on some pages within the control panel. This should be consistent and reflect the value of the `site` param in the control panel query string.

For example you can navigate to "globals" where the body class "site handle" value matches the query string "site param" but if you then go to the "entries" section (where the currently selected site is reflected in the query string) the "site handle" disappears from the body class.

It would be great if the "site handle" could be applied to the body class of all control panel to mimic the value reflected by the "site" param in the query string.

**Comment by brandonkelly** [comment, marked as answer]
Good call. Just added that for the next release (77f87cc59ff3cc9a2e49434d91715eaa3c3123d1). The class is automatically updated when you change the selected site on element index pages as well.

**Reply by brandonkelly** [reply]
Craft 4.0.4 is out now with that change ✨
